homeassistant/components/alarm_control_panel/ifttt.py

```diff
@@ -0,0 +1,154 @@
+"""
+Interfaces with alarm control panels that have to be controlled through IFTTT.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.ifttt/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN, PLATFORM_SCHEMA)
+from homeassistant.components.ifttt import (
+    ATTR_EVENT, DOMAIN as IFTTT_DOMAIN, SERVICE_TRIGGER)
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_STATE, CONF_NAME, CONF_CODE,
+    STATE_ALARM_DISARMED, STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMED_HOME, STATE_ALARM_ARMED_AWAY)
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = ['ifttt']
+
+_LOGGER = logging.getLogger(__name__)
+
+ALLOWED_STATES = [
+    STATE_ALARM_DISARMED, STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMED_AWAY, STATE_ALARM_ARMED_HOME]
+
+DATA_IFTT_ALARM = 'ifttt_alarm'
+DEFAULT_NAME = "Home"
+
+EVENT_ALARM_ARM_AWAY = "alarm_arm_away"
+EVENT_ALARM_ARM_HOME = "alarm_arm_home"
+EVENT_ALARM_ARM_NIGHT = "alarm_arm_night"
+EVENT_ALARM_DISARM = "alarm_disarm"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_CODE): cv.string,
+})
+
+SERVICE_PUSH_ALARM_STATE = "push_alarm_state"
```

Comment:
Prefix platform specific services with the platform name. So this should be `ifttt_push_alarm_state`